Phonebook.Frontend/src/app/services/api/person.service.ts

```diff
@@ -102,15 +103,12 @@ export class PersonService {
     );
   }
 
-  public getPersonsByRoom(positionArray: string[]): Observable<Person[]> {
+  public getPersonsByRoom(roomId: string): Observable<Person[]> {
```

Comment:
gleiches problem roomId war vorher nicht unique

Reply:
is now unique